turbo/engineapi/engine_types/jsonrpc.go

```diff
@@ -44,6 +44,7 @@ type PayloadAttributes struct {
 	PrevRandao            common.Hash         `json:"prevRandao"            gencodec:"required"`
 	SuggestedFeeRecipient common.Address      `json:"suggestedFeeRecipient" gencodec:"required"`
 	Withdrawals           []*types.Withdrawal `json:"withdrawals"`
+	ParentBeaconBlockRoot common.Hash         `json:"parentBeaconBlockRoot" gencodec:"required"`
```

Comment:
Probably shouldn't have `gencodec:"required"` since it's not present before Cancun.